README.md

```diff
@@ -1,8 +1,10 @@
-[![Tests workflow](https://github.com/redhat-developer/vscode-quarkus/actions/workflows/tests.yml/badge.svg?branch=master)](https://github.com/redhat-developer/vscode-quarkus/actions)
-[![Marketplace Version](https://vsmarketplacebadge.apphb.com/version/redhat.vscode-quarkus.svg "Current Release")](https://marketplace.visualstudio.com/items?itemName=redhat.vscode-quarkus)
-
 # Quarkus Tools for Visual Studio Code
 
+[![Visual Studio Marketplace](https://img.shields.io/visual-studio-marketplace/v/redhat.vscode-quarkus?style=for-the-badge&label=VS%20Marketplace&logo=visual-studio-code)](https://marketplace.visualstudio.com/items?itemName=redhat.vscode-quarkus)
+[![Installs](https://img.shields.io/visual-studio-marketplace/i/redhat.vscode-quarkus?style=for-the-badge)](https://marketplace.visualstudio.com/items?itemName=redhat.vscode-quarkus)
+[![Build Status](https://img.shields.io/github/workflow/status/redhat-developer/vscode-quarkus/Tests?style=for-the-badge&logo=github)](https://github.com/redhat-developer/vscode-quarkus/actions)
```

Comment:
For the link for the build status, I think it would be better to include the filter to only show the master branch CI runs, eg.

https://github.com/redhat-developer/vscode-quarkus/actions?query=branch%3Amaster